### PR TITLE
Emily/pagination styling

### PR DESCRIPTION
**Pull request by emilyzhong**
## Pagination Styling
Styling for pagination navigation
https://app.asana.com/0/860297512787618/1112571034339612/f
(Sorry I didn't see your comment about the backend until now but all the styling is there if you wanna throw it in)

### Related PRs
https://github.com/calblueprint/SFAI/pull/75

### Migrations
None

### Tests Performed, Edge Cases
None 

### Screenshots
image

CC: @by-co

### CODE REVIEW

File: app/assets/stylesheets/works.scss [hunk 1]
```diff
@@ -53,3 +53,24 @@
     }
   }
 }
+
+#pagination{
```
Comment (by-co) [inline app/assets/stylesheets/works.scss]: Hmm, can we move this to a higher level .scss file? Eventually want to apply `ReactPagination` to the `AllArtists` page too, cc @tommypoa

File: app/assets/stylesheets/works.scss [hunk 2]
```diff
@@ -53,3 +53,24 @@
...
+  li {
+    width: 32px;
+    height: 32px;
+    text-align: center;
+    line-height: 32px;
+    margin-left: 8px;
+    background-color: white;
+
+    &.active {
+      background-color: $magenta;
```
Comment (by-co) [inline app/assets/stylesheets/works.scss]: Let's keep this `magenta` styling, though, for the works page in particular

File: app/assets/stylesheets/works.scss [hunk 3]
```diff
@@ -53,3 +53,24 @@
...
+  }
+
+  li {
+    width: 32px;
+    height: 32px;
+    text-align: center;
+    line-height: 32px;
+    margin-left: 8px;
+    background-color: white;
+
```
Comment (by-co) [inline app/assets/stylesheets/works.scss]: Can we add `cursor: pointer` effect too for hover? :)

**Comment by emilyzhong** [comment]
💖 :')

**Comment by emilyzhong** [comment]
💓 :')
